index.html

```diff
@@ -5,5 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <main>
+      <div>
```

Comment:
- Semantic tags - main -> section -> article -> h1
- Add class="question_card" to the article tag